app/lib/manage.py

```diff
@@ -49,26 +73,58 @@ def joinThreads(threads: List[threading.Thread]):
 appDataDir = os.path.join(nodeRoot, "app-data")
 userFile = os.path.join(nodeRoot, "db", "user.json")
 legacyScript = os.path.join(nodeRoot, "scripts", "app")
-
-# Returns a list of every argument after the second one in sys.argv joined into a string by spaces
+with open(os.path.join(nodeRoot, "db", "dependencies.yml"), "r") as file: 
+  dependencies = yaml.safe_load(file)
 
 
+# Returns a list of every argument after the second one in sys.argv joined into a string by spaces
 def getArguments():
     arguments = ""
     for i in range(3, len(argv)):
         arguments += argv[i] + " "
     return arguments
 
+def handleAppV4(app):
+    composeFile = os.path.join(appsDir, app, "docker-compose.yml")
+    os.chown(os.path.join(appsDir, app), 1000, 1000)
+    os.system("docker run --rm -v {}:/apps -u 1000:1000 {} /app-cli convert --app-name '{}' --port-map /apps/ports.json /apps/{}/app.yml /apps/{}/result.yml --services 'lnd'".format(appsDir, dependencies['app-cli'], app, app, app))
```

Comment:
This pulls the image for the `app-cli` and runs containers for every v4 app I believe. It works but might lead to conflicts, or at least room for speed a improvement.

```
Unable to find image 'ghcr.io/runcitadel/app-cli:main@sha256:f532923eac28cfac03579cbb440397bcf16c8730f291b39eeada8278331f7054' locally
Unable to find image 'ghcr.io/runcitadel/app-cli:main@sha256:f532923eac28cfac03579cbb440397bcf16c8730f291b39eeada8278331f7054' locally
Unable to find image 'ghcr.io/runcitadel/app-cli:main@sha256:f532923eac28cfac03579cbb440397bcf16c8730f291b39eeada8278331f7054' locally
Unable to find image 'ghcr.io/runcitadel/app-cli:main@sha256:f532923eac28cfac03579cbb440397bcf16c8730f291b39eeada8278331f7054' locally
Unable to find image 'ghcr.io/runcitadel/app-cli:main@sha256:f532923eac28cfac03579cbb440397bcf16c8730f291b39eeada8278331f7054' locally
Unable to find image 'ghcr.io/runcitadel/app-cli:main@sha256:f532923eac28cfac03579cbb440397bcf16c8730f291b39eeada8278331f7054' locally
Unable to find image 'ghcr.io/runcitadel/app-cli:main@sha256:f532923eac28cfac03579cbb440397bcf16c8730f291b39eeada8278331f7054' locally
Unable to find image 'ghcr.io/runcitadel/app-cli:main@sha256:f532923eac28cfac03579cbb440397bcf16c8730f291b39eeada8278331f7054' locally
ghcr.io/runcitadel/app-cli@sha256:f532923eac28cfac03579cbb440397bcf16c8730f291b39eeada8278331f7054: Pulling from runcitadel/app-cli
ghcr.io/runcitadel/app-cli@sha256:f532923eac28cfac03579cbb440397bcf16c8730f291b39eeada8278331f7054: Pulling from runcitadel/app-cli
ghcr.io/runcitadel/app-cli@sha256:f532923eac28cfac03579cbb440397bcf16c8730f291b39eeada8278331f7054: Pulling from runcitadel/app-cli
ghcr.io/runcitadel/app-cli@sha256:f532923eac28cfac03579cbb440397bcf16c8730f291b39eeada8278331f7054: Pulling from runcitadel/app-cli
ghcr.io/runcitadel/app-cli@sha256:f532923eac28cfac03579cbb440397bcf16c8730f291b39eeada8278331f7054: Pulling from runcitadel/app-cli
ghcr.io/runcitadel/app-cli@sha256:f532923eac28cfac03579cbb440397bcf16c8730f291b39eeada8278331f7054: Pulling from runcitadel/app-cli
ghcr.io/runcitadel/app-cli@sha256:f532923eac28cfac03579cbb440397bcf16c8730f291b39eeada8278331f7054: Pulling from runcitadel/app-cli
ghcr.io/runcitadel/app-cli@sha256:f532923eac28cfac03579cbb440397bcf16c8730f291b39eeada8278331f7054: Pulling from runcitadel/app-cli
```